multimodal/tests/unittests/test_backward_compatibility.py

```diff
@@ -0,0 +1,67 @@
+import os
+import shutil
+import tempfile
+import numpy.testing as npt
+import pytest
+
+from autogluon.multimodal.constants import BINARY, MULTICLASS
+from autogluon.multimodal import MultiModalPredictor, AutoMMPredictor
+from datasets import AmazonReviewSentimentCrossLingualDataset
+from utils import (
+    download,
+    protected_zip_extraction,
+    get_home_dir,
+)
+
+
+def verify_predictor_save_load(predictor, df, verify_embedding=True, cls=MultiModalPredictor):
```

Comment:
I guess we can just do this:

`from test_predictor import verify_predictor_save_load`